### PR TITLE
Make building clients for other architectures optional

### PR DESCRIPTION
**Pull request by tdawson**
This is in preparation for building this package in Fedora.

### CODE REVIEW

File: origin.spec [hunk 1]
```diff
@@ -22,6 +22,9 @@
 %global ldflags -X github.com/openshift/origin/pkg/version.majorFromGit 0 -X github.com/openshift/origin/pkg/version.minorFromGit 0+ -X github.com/openshift/origin/pkg/version.versionFromGit v0.0.1 -X github.com/openshift/origin/pkg/version.commitFromGit 86b5e46 -X k8s.io/kubernetes/pkg/version.gitCommit 6241a21 -X k8s.io/kubernetes/pkg/version.gitVersion v0.11.0-330-g6241a21
 }
 
+# Make building clients for other architectures optional
+%global make_redistributable 1
```
Comment (sdodson) [inline origin.spec]: Is there something in the fedora build process that will set this to 0?

**Comment by danmcp** [comment]
What's the priority on this?

@sdodson Can you review again please.

**Comment by sdodson** [comment]
@danmcp low. It's low risk but lets wait until after 1.1.

**Comment by sdodson** [comment]
LGTM :+1: wrt to review

**Comment by tdawson** [comment]
[test]

**Comment by sdodson** [comment]
LGTM

**Comment by tdawson** [comment]
[merge]

**Comment by tdawson** [comment]
I have tested the resulting src.rpm in epel, fedora, rhel-local, and rhel-brew.  It build what you would expect it to do.
epel and fedora - no clients-redistributable built
rhel-local and rhel-brew - yes  clients-redistributable was built.

**Pull request merged**